readme.txt

```diff
@@ -129,6 +129,8 @@ If you get stuck, you can ask for help in the Plugin Forum.
 == Changelog ==
 
 = 8.8.0 - xxxx-xx-xx =
+* Add - Implemented the "Update all subscriptions payment methods" checkbox on My Account → Payment methods for UPE payment methods.
+* Add - Add support for the new Stripe Checkout Element on the shortcode checkout page. 
```

Comment:
This changelog entry wasn't add as part of: https://github.com/woocommerce/woocommerce-gateway-stripe/pull/3436 so I added it when fixing the conflicts